tests/Load/config.sh

```
@@ -0,0 +1,33 @@
+  read -r -p "$prompt_message [$default_value]: " user_input
+  echo "${user_input:-$default_value}"
+}
+
+export AWS_PAGER=""
+export REGION=$(prompt_for_input "Enter AWS Region" "us-east-1")
+export AMI_ID=$(prompt_for_input "Enter AMI ID" "ami-0e86e20dae9224db8")
+export INSTANCE_TYPE=$(prompt_for_input "Enter EC2 instance type" "t2.micro")
+export INSTANCE_TAG=$(prompt_for_input "Enter instance tag" "LoadTestInstance")
+export ROLE_NAME=$(prompt_for_input "Enter IAM Role name" "EC2S3WriteAccessRole")
+export BRANCH_NAME=$(prompt_for_input "Enter Git branch name" "main")
+export BUCKET_NAME="loadtest-bucket-$(uuidgen)"
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)